internal/actions/runWorkflow.go

```diff
@@ -265,6 +272,27 @@ func finalize(inputs finalizeInputs) error {
 			}
 		}
 
+		inputs.Outputs["commit_hash"] = commitHash
+	case environment.ModeTest:
```

Comment:
I thought we were exiting before calling finalize? Line 169

Reply:
Ya true I changed that later. This is redundant